apiml-common/src/main/java/com/ca/mfaas/product/logging/annotations/InjectApimlLogger.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.product.logging.annotations;
+
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * After enabling apiml logging {@link EnableApimlLogger} it can be used as an injector.
```

Comment:
Maybe, "If you enable APIML logging, you can use {@link EnableApimlLogger} as an injector". Does that make sense, or did I get it wrong?

Reply:
After enabling APIML logging with  {@link EnableApimlLogger}, the InjectApimlLogger can be used as an injector.

Reply:
Perfect! "When you enable APIML logging with {@link EnableApimlLogger}, you can use EnableApimlLogger as an injector"